data/yam/agama/auto/lib/questions.libsonnet

```diff
@@ -0,0 +1,12 @@
+{
+    questions_decrypt(decrypt_password=''):: {
```

Comment:
the library already is called questions, so ` luks_activation(password=''):: {` should do.